.github/ISSUE_TEMPLATE/new-bug.md

```diff
@@ -0,0 +1,37 @@
+---
+name: 🐛 Bug Report 
+about: Create a report for any found bugs.
+labels: 'Type: Bug'
+title: "🐛 "
+---
+
+<!-- **Tip:** Delete parts that are not relevent -->
+
+### 🐛 Desrcibe the Bug
+<!-- A clear and concise description of what the bug is -->
+
```

Comment:
Add a priority level.

Reply:
I believe priority level should be a label rather than in the content of the issue